src/components/shared/thread/ActivityGraph.js

```diff
@@ -145,7 +145,10 @@ class ThreadActivityGraphImpl extends React.PureComponent<Props, State> {
 
   _onClick = (event: SyntheticMouseEvent<HTMLCanvasElement>) => {
     const sampleState = this._getSampleAtMouseEvent(event);
-    this.props.onSampleClick(event, sampleState ? sampleState.sample : null);
+    this.props.onSampleClick(
+      event,
+      sampleState && sampleState.sample ? sampleState.sample : null
+    );
```

Comment:
optional: this change may not be necessary: indeed, if `sampleState.sample` is null, `null` would be used anyway.